Dockerfile-56

```diff
@@ -125,5 +125,5 @@ RUN cp /etc/php/7.0/mods-available/* $CONF_PHPMODS && \
     # - Run standard set of tweaks to ensure runs performant, reliably, and consistent between variants \
     /bin/bash -e /prep-php.sh
 
-RUN goss -g /tests/php-fpm/legacy.goss.yaml validate && \
+RUN goss -g /tests/php-fpm/56.goss.yaml validate && \
```

Comment:
hm...I wonder what's wrong in git. That's been updated (for all of these) in my branch